.gitignore

```diff
@@ -1 +1,2 @@
 build/
+.idea/
```

Comment:
I do not use jetbrains.  There is no reason for my git to have to ignore that path.  It's better practice to put ignores related to your personal environment in your personal ignore file, `.git/info/exclude`.

Reply:
Sounds good to me